mlir/lib/Dialect/SparseTensor/IR/Detail/LvlTypeParser.cpp

```diff
@@ -41,8 +41,16 @@ FailureOr<uint64_t> LvlTypeParser::parseLvlType(AsmParser &parser) const {
     ParseResult res = parser.parseCommaSeparatedList(
         mlir::OpAsmParser::Delimiter::OptionalSquare,
         [&]() -> ParseResult { return parseStructure(parser, &structure); },
-        " in block n out of m");
+        " in structure n out of m");
```

Comment:
I notice this just now, but can we s/structure/structured/g for all.
or perhaps structured_n_m 
(but calling this structured rather than structure feels a bit more consistent(